src/Serialization/main.jl

```diff
@@ -186,6 +190,12 @@ function save_type_params(s::SerializerState, obj::Any, key::Symbol)
   save_type_params(s, obj)
 end
 
+function save_attrs(s::SerializerState, obj::T) where T
```

Comment:
This and `load_attrs` should be added to the `using Oscar:...` list here (https://github.com/oscar-system/Oscar.jl/pull/4021/files#diff-a00d43318a99f7f3073a8eeebdf4de11fe5c61405e45cd918241c100b82bed64R390) once https://github.com/oscar-system/Oscar.jl/pull/4021 is merged